internal/storage/caching/fast_stat_bucket_test.go

```diff
@@ -127,7 +126,7 @@ func (t *CreateObjectTest) WrappedSucceeds() {
 		WillOnce(Return(obj, nil))
 
 	// Insert
-	ExpectCall(t.cache, "Insert")(obj, timeutil.TimeEq(t.clock.Now().Add(ttl)))
+	ExpectCall(t.cache, "Insert")(Any(), timeutil.TimeEq(t.clock.Now().Add(ttl)))
```

Comment:
With the current implementation, we can't because min object is created in the CreateObject/CopyObject/ComposeObject/UpdateObject methods by converting object to min object and passed on to Insert.